### ISSUE TITLE
 ERROR: Could not find OpenVPN configuration "CA Toronto" for provider PIA

### ISSUE THREAD

**Issue by Grzy7316** [haugene/docker-transmission-openvpn#1470]
I am using this container in portainer- Under Envrionment variables, I have the following : 
OPENVPN_PROVIDER = PIA
OPENVPN_CONFIG = CA Toronto

**Comment by jkwill87** [comment]
Possibly related to #1463 and this update on the PIA site:
![image](https://user-images.githubusercontent.com/4343678/98406897-c169e000-203c-11eb-89d1-20951e9634a5.png)

Either way, I'm having the same issue too.

**Comment by RickyXwang** [comment]
Can you post your log and start config/ docker-compose? Issue 1463 is due to AUTH_FAILED, not error `Could not find OpenVPN configuration`

**Comment by cloudynetwork** [comment]
I receive the same error for any other PIA locations I have tried eg: 

"Supplied config "France".ovpn could not be found.

Container restarts with this error.

**Comment by ilike2burnthing** [comment]
@cloudynetwork remove the quotation marks from around the location.

**Comment by RickyXwang** [comment]
Without log/configeation I can't tell what is wrong.

**Comment by haugene** [comment]
@RickyXwang and @ilike2burnthing are right. This is about quoting your server names, don't. And provide logs if you want help with debugging.

FYI: I just tested and got AUTH_FAILED twice from CA Toronto, but successfully connected to CA Montreal with the same credentials. So please test a few different servers as well. France has been working fine for me this far though.

**Comment by cloudynetwork** [comment]
@haugene @ilike2burnthing Thanks, removing the quotes worked.....but I've had the quotes in the config file since I created the config at least 12 months ago. Clearly having quotes there is incorrect...but it broken suddenly. Since it's wrong I'm not going to suggest otherwise, but for whatever reason it worked until recently....

**Comment by haugene** [comment]
Yes. It broke due to some changes in how we handle environment variables going from version 2.x and over to 3.0.

I tried to put a warning on the front page: https://github.com/haugene/docker-transmission-openvpn
> Version 3.0 released - we have some breaking changes (but not much)

I'll be honest and say that I didn't know about this breaking change explicitly when I released 3.0. But it is a bigger rewrite and it's gone relatively smooth so far so thanks for your patience :+1:
